### DISCUSSION TITLE
SOC `PROCAR` Parsing

### DISCUSSION THREAD

**Discussion by kavanase** [materialsproject/pymatgen#3722] [General]
Not entirely sure where to post this, but just wanted to give an fyi that [`easyunfold`](https://smtg-bham.github.io/easyunfold/) has a VASP `PROCAR` parser that can handle SOC and different VASP versions, as well as combining information from multiple `PROCAR`s (e.g. from a split band structure calculation), which the `pymatgen` currently can't do. Also with quite time-efficient parsing.
Just in case this was desired functionality at any point, would be useful to know!

The object architecture is quite similar to the `pymatgen` one (just with `proj_data` instead of `data` as attributes etc).

**Comment by janosh** [comment]
this sounds great! I think i speak for everyone when i say this would be a very welcome addition to `pymatgen`, especially if it can be made backwards compatible. i'd be happy to help get a PR merged

**Comment by kavanase** [comment]
Finally got round to this 😅 
https://github.com/materialsproject/pymatgen/pull/3890
